### DISCUSSION TITLE
No Journal Abbreviation Styles in the BBT dropdown menu

### DISCUSSION THREAD

**Discussion by ht145** [retorquere/zotero-better-bibtex#3056] [General]
As mentioned in the title, there don't seem to be any journal abbreviation styles in the drop-down menu in the bbt  preferences pane. Does anyone have a solution for this?
![image](https://github.com/user-attachments/assets/6242e9a0-485c-42dc-86d5-94193c91b3d9)

**Comment by retorquere** [comment]
That's only supposed to show on Juris-M. Zotero does not support this.

**Comment by ht145** [comment]
Got it. Thanks for letting me know.
